Task_DEV_5/Text2.txt

```diff
@@ -0,0 +1,7 @@
+mothfather
```

Comment:
Do we need this file? I don't see places where it is using.

Reply:
Done.